<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1940</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>組み込みシステムを用いた産業用画像検出システムの設計（日本語版）@by Fan</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="connectBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>431</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>ネットワークポート</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="IPLabel">
         <property name="text">
          <string>IP：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtIP">
         <property name="text">
          <string>192.168.1.30</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="portLabel">
         <property name="text">
          <string>ポート番号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtPort">
         <property name="text">
          <string>8088</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connectBtn">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>つながる</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="txtReceive"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="edgeImageGB">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>0</y>
      <width>641</width>
      <height>511</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>处理结果</string>
    </property>
    <widget class="QLabel" name="resultLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="doGB">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>520</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>处理</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="photoBtn">
         <property name="text">
          <string>写真を撮る</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveFormerButton">
         <property name="text">
          <string>元画像を保存する</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="edgeBtn">
         <property name="text">
          <string>検出</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveBtn">
         <property name="text">
          <string>結果を保存する</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="photoImageGB">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>641</width>
      <height>511</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>写真</string>
    </property>
    <widget class="QLabel" name="photoLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="getImageGB">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>511</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>ビデオカメラ</string>
    </property>
    <widget class="QLabel" name="showLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="setGB">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>520</y>
      <width>331</width>
      <height>184</height>
     </rect>
    </property>
    <property name="title">
     <string>シリアルポート</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="serialNumlabel">
           <property name="text">
            <string>番号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="PortBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="baudRatelabel">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>速度(bps)：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="BaudBox">
           <property name="currentText">
            <string>921600</string>
           </property>
           <item>
            <property name="text">
             <string>921600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>256000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1200</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="datalabel">
           <property name="text">
            <string>データビット：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="BitNumBox">
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="checklabel">
           <property name="text">
            <string>パリティ：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ParityBox">
           <item>
            <property name="text">
             <string>無し</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>奇数</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>偶数</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="stoplabel">
           <property name="text">
            <string>ストップビット：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="StopBox">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1.5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="serialOpenButton">
         <property name="text">
          <string>ポート開く</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="focusButton">
         <property name="text">
          <string>焦点合わせ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="beepButton">
         <property name="text">
          <string>ブザー開く</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ledCouldButton">
         <property name="text">
          <string>スピードライト開く</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="parameterGroupBox">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>520</y>
      <width>391</width>
      <height>178</height>
     </rect>
    </property>
    <property name="title">
     <string>パラメータ</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="edgeLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>しきい値：20</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="edgeSlider">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>120</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="realLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>カップの実際な輪郭:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reallineEdit">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>26.8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cmLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>cm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="deltaLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>偏り：</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="deltaLineEdit">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="minlengthLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>傷の最小値：</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="minlengthLineEdit">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cmLabel_2">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>cm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="resultGB">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>520</y>
      <width>471</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>结果</string>
    </property>
    <widget class="QLabel" name="resolutionLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>331</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>解像度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="finallengthLabel">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>120</y>
       <width>331</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>傷の長さ：</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1940</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>edit</string>
    </property>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenImage">
   <property name="text">
    <string>open a picture</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>clear out</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>It is a japnese version</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
